docs-chef-io/content/server/upgrades.md

```diff
@@ -393,7 +405,8 @@ Check the [post upgrade steps](#post-upgrade-steps) if you are upgrading from a
    sudo systemctl start postgresql.service
    ```
 
-1. Log in as the postgres user and confirm that the new PostgreSQL version is 13.3.
+1. Log in as the postgres user and confirm that the new PostgreSQL version is correct.
```

Comment:
```suggestion
1. Log in as the Postgres user and confirm that the new PostgreSQL version is correct.
```

Reply:
"Customarily, this user will be named postgres" from https://www.postgresql.org/docs/8.0/user-manag.html

I think we want:

```
Log in as the `postgres` user and confirm that the new PostgreSQL version is correct.
```

Reply:
Whoops, good catch.

Reply:
Caught another one also.